wgs/wgs-gather-vcfs/wgs-gather-vcfs.wdl

```diff
@@ -0,0 +1,169 @@
+## wgs-gather-vcfs.wdl
+## 
+## This workflow runs the GATK's `GatherVcfsCloud` on a GCS directory containing sharded VCF files
+## from a joint genotyping workflow such as the one here: https://github.com/gatk-workflows/gatk4-germline-snps-indels.
+##
+## Each VCF file corresponds to an interval from the joint genotyping intervals file. That file is a required
+## input to this workflow.
+##
+## The output of the workflow is 1 VCF file per chromosome requested.
+##
+## Inputs:
+## - intervals_file: GCS path of intervals file
+## - chromosomes: An array of chromosome to gather, such as ["1", "2", "21", "22", "X", "Y"]
+## - shards_to_ignore: An array of numbers from the intervals file to ignore, ex [576, 1032]
```

Comment:
What was the motivation for shards_to_ignore? That should be explained here.

Reply:
Added in comments